tests/functional/report_viewer_api/test_get_run_results.py

```diff
@@ -229,7 +190,7 @@ def test_get_run_results_sorted2(self):
                                           [])
         self.assertIsNotNone(run_results)
 
-        print_run_results(run_results)
+        # print_run_results(run_results)
```

Comment:
Comments removed. With `.noserc` it is configurable what will be printed to stdout.